<!DOCTYPE html>
<html dir="ltr" lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<meta http-equiv="Content-Language" content="en" />
<title>ncurses.patch - crux-ports - Ports for the CRUX Linux distro.
</title>
<link rel="icon" type="image/png" href="../../favicon.png" />
<link rel="alternate" type="application/atom+xml" title="crux-ports.git Atom Feed" href="../../atom.xml" />
<link rel="stylesheet" type="text/css" href="../../style.css" />
</head>
<body>
<table><tr><td><a href="../../../"><img src="../../logo.png" alt="" width="32" height="32" /></a></td><td><h1>crux-ports</h1><span class="desc">Ports for the CRUX Linux distro.
</span></td></tr><tr class="url"><td></td><td>git clone <a href="git://thetudor.ddns.net/crux-ports.git">git://thetudor.ddns.net/crux-ports.git</a></td></tr><tr><td></td><td>
<a href="../../log.html">Log</a> | <a href="../../files.html">Files</a> | <a href="../../refs.html">Refs</a></td></tr></table>
<hr/>
<div id="content">
<p> ncurses.patch (3108B)</p><hr/><table id="blob"><tr><td class="num"><pre>
<a href="#l1" id="l1">1</a>
<a href="#l2" id="l2">2</a>
<a href="#l3" id="l3">3</a>
<a href="#l4" id="l4">4</a>
<a href="#l5" id="l5">5</a>
<a href="#l6" id="l6">6</a>
<a href="#l7" id="l7">7</a>
<a href="#l8" id="l8">8</a>
<a href="#l9" id="l9">9</a>
<a href="#l10" id="l10">10</a>
<a href="#l11" id="l11">11</a>
<a href="#l12" id="l12">12</a>
<a href="#l13" id="l13">13</a>
<a href="#l14" id="l14">14</a>
<a href="#l15" id="l15">15</a>
<a href="#l16" id="l16">16</a>
<a href="#l17" id="l17">17</a>
<a href="#l18" id="l18">18</a>
<a href="#l19" id="l19">19</a>
<a href="#l20" id="l20">20</a>
<a href="#l21" id="l21">21</a>
<a href="#l22" id="l22">22</a>
<a href="#l23" id="l23">23</a>
<a href="#l24" id="l24">24</a>
<a href="#l25" id="l25">25</a>
<a href="#l26" id="l26">26</a>
<a href="#l27" id="l27">27</a>
<a href="#l28" id="l28">28</a>
<a href="#l29" id="l29">29</a>
<a href="#l30" id="l30">30</a>
<a href="#l31" id="l31">31</a>
<a href="#l32" id="l32">32</a>
<a href="#l33" id="l33">33</a>
<a href="#l34" id="l34">34</a>
<a href="#l35" id="l35">35</a>
<a href="#l36" id="l36">36</a>
<a href="#l37" id="l37">37</a>
<a href="#l38" id="l38">38</a>
<a href="#l39" id="l39">39</a>
<a href="#l40" id="l40">40</a>
<a href="#l41" id="l41">41</a>
<a href="#l42" id="l42">42</a>
<a href="#l43" id="l43">43</a>
<a href="#l44" id="l44">44</a>
<a href="#l45" id="l45">45</a>
<a href="#l46" id="l46">46</a>
<a href="#l47" id="l47">47</a>
<a href="#l48" id="l48">48</a>
<a href="#l49" id="l49">49</a>
<a href="#l50" id="l50">50</a>
<a href="#l51" id="l51">51</a>
<a href="#l52" id="l52">52</a>
<a href="#l53" id="l53">53</a>
<a href="#l54" id="l54">54</a>
<a href="#l55" id="l55">55</a>
<a href="#l56" id="l56">56</a>
<a href="#l57" id="l57">57</a>
<a href="#l58" id="l58">58</a>
<a href="#l59" id="l59">59</a>
<a href="#l60" id="l60">60</a>
<a href="#l61" id="l61">61</a>
<a href="#l62" id="l62">62</a>
<a href="#l63" id="l63">63</a>
<a href="#l64" id="l64">64</a>
<a href="#l65" id="l65">65</a>
<a href="#l66" id="l66">66</a>
<a href="#l67" id="l67">67</a>
<a href="#l68" id="l68">68</a>
<a href="#l69" id="l69">69</a>
<a href="#l70" id="l70">70</a>
<a href="#l71" id="l71">71</a>
<a href="#l72" id="l72">72</a>
<a href="#l73" id="l73">73</a>
<a href="#l74" id="l74">74</a>
<a href="#l75" id="l75">75</a>
<a href="#l76" id="l76">76</a>
<a href="#l77" id="l77">77</a>
<a href="#l78" id="l78">78</a>
<a href="#l79" id="l79">79</a>
<a href="#l80" id="l80">80</a>
<a href="#l81" id="l81">81</a>
<a href="#l82" id="l82">82</a>
<a href="#l83" id="l83">83</a>
<a href="#l84" id="l84">84</a>
<a href="#l85" id="l85">85</a>
<a href="#l86" id="l86">86</a>
<a href="#l87" id="l87">87</a>
<a href="#l88" id="l88">88</a>
<a href="#l89" id="l89">89</a>
<a href="#l90" id="l90">90</a>
</pre></td><td><pre>
diff -wbBur gtypist-2.9.4/configure.ac gtypist-2.9.4.my/configure.ac
--- gtypist-2.9.4/configure.ac	2014-02-02 15:38:40.000000000 +0400
+++ gtypist-2.9.4.my/configure.ac	2014-02-03 13:08:53.085983684 +0400
@@ -44,20 +44,20 @@
 AC_FUNC_STRTOD
 AC_CHECK_FUNCS([__argz_count __argz_next __argz_stringify dcgettext getcwd getwd mempcpy memset munmap nl_langinfo setlocale stpcpy strcasecmp strchr strcspn strdup strstr strtoul])
 
-# check for libncursesw
+# check for libncurses
 
-AC_CHECK_HEADER(ncursesw/ncurses.h, HAVE_NCURSESW_H=1)
-AC_CHECK_LIB(ncursesw, add_wch, HAVE_LIBNCURSESW=1)
+AC_CHECK_HEADER(ncurses.h, HAVE_NCURSESW_H=1)
+AC_CHECK_LIB(ncurses, add_wch, HAVE_LIBNCURSESW=1)
 if test -n &quot;$HAVE_NCURSESW_H&quot; -a -n &quot;$HAVE_LIBNCURSESW&quot;;  then
-   LIBS=&quot;-lncursesw $LIBS&quot;
+   LIBS=&quot;-lncurses $LIBS&quot;
 else
-   echo -e &quot;Error:  both library and header files for the ncursesw library\n&quot;\
+   echo -e &quot;Error:  both library and header files for the ncurses library\n&quot;\
        &quot;are required to build this package.  See INSTALL file for&quot;\
        &quot;further information. On Debian/Ubuntu you need to install libncursesw5-dev.&quot;
    exit 1;
 fi
 AC_SEARCH_LIBS(cbreak, tinfo, [],
-  [AC_MSG_ERROR([Can&apos;t find cbreak() in -lncursesw or -ltinfo])])
+  [AC_MSG_ERROR([Can&apos;t find cbreak() in -lncurses or -ltinfo])])
 
 
 # iconv
diff -wbBur gtypist-2.9.4/src/cursmenu.c gtypist-2.9.4.my/src/cursmenu.c
--- gtypist-2.9.4/src/cursmenu.c	2013-08-18 18:36:14.000000000 +0400
+++ gtypist-2.9.4.my/src/cursmenu.c	2014-02-03 13:09:04.669316878 +0400
@@ -24,7 +24,7 @@
 #ifdef HAVE_PDCURSES
 #include &lt;curses.h&gt;
 #else
-#include &lt;ncursesw/ncurses.h&gt;
+#include &lt;ncurses.h&gt;
 #endif
 
 #include &quot;error.h&quot;
diff -wbBur gtypist-2.9.4/src/error.c gtypist-2.9.4.my/src/error.c
--- gtypist-2.9.4/src/error.c	2013-08-18 18:36:14.000000000 +0400
+++ gtypist-2.9.4.my/src/error.c	2014-02-03 13:09:04.669316878 +0400
@@ -25,7 +25,7 @@
 #ifdef HAVE_PDCURSES
 #include &lt;curses.h&gt;
 #else
-#include &lt;ncursesw/ncurses.h&gt;
+#include &lt;ncurses.h&gt;
 #endif
 
 #include &lt;stdlib.h&gt;
diff -wbBur gtypist-2.9.4/src/gtypist.c gtypist-2.9.4.my/src/gtypist.c
--- gtypist-2.9.4/src/gtypist.c	2013-08-18 18:36:14.000000000 +0400
+++ gtypist-2.9.4.my/src/gtypist.c	2014-02-03 13:09:04.669316878 +0400
@@ -31,7 +31,7 @@
 #ifdef HAVE_PDCURSES
 #include &lt;curses.h&gt;
 #else
-#include &lt;ncursesw/ncurses.h&gt;
+#include &lt;ncurses.h&gt;
 #endif
 
 #include &lt;time.h&gt;
diff -wbBur gtypist-2.9.4/src/script.c gtypist-2.9.4.my/src/script.c
--- gtypist-2.9.4/src/script.c	2013-08-18 18:36:14.000000000 +0400
+++ gtypist-2.9.4.my/src/script.c	2014-02-03 13:09:04.669316878 +0400
@@ -24,7 +24,7 @@
 #ifdef HAVE_PDCURSES
 #include &lt;curses.h&gt;
 #else
-#include &lt;ncursesw/ncurses.h&gt;
+#include &lt;ncurses.h&gt;
 #endif
 
 #include &quot;error.h&quot;
diff -wbBur gtypist-2.9.4/src/utf8.c gtypist-2.9.4.my/src/utf8.c
--- gtypist-2.9.4/src/utf8.c	2013-08-18 18:36:14.000000000 +0400
+++ gtypist-2.9.4.my/src/utf8.c	2014-02-03 13:09:04.669316878 +0400
@@ -23,7 +23,7 @@
 #ifdef HAVE_PDCURSES
 #include &lt;curses.h&gt;
 #else
-#include &lt;ncursesw/ncurses.h&gt;
+#include &lt;ncurses.h&gt;
 #endif
 
 #include &lt;stdlib.h&gt;
</pre></td></tr></table>
</div>
</body>
</html>
